### PR TITLE
Extended to display more fields and relationships

### PR DESCRIPTION
**Pull request by nwrigh**
Ability to display all the additional fields against a concept smartlogic. Plus the narrower, broader and related concepts for a Thing

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -61,18 +62,43 @@ func main() {
...
+		"NEO_URL":              *neoURL,
+		"LOG_LEVEL":            *logLevel,
+	}).Info("Starting app with arguments")
+
 	log.Infof("Application started with args %s", os.Args)
 	app.Run(os.Args)
 }
 
-func runServer(neoURL string, port string, cacheDuration string, env string) {
+func runServer(neoURL string, port string, cacheDuration string, healthcheckInterval string, env string) {
```
Comment (scott-ace-newton) [inline app.go]: You dont do anything with this?

File: things/cypher.go [hunk 2]
```diff
@@ -30,54 +30,90 @@ func (cd cypherDriver) checkConnectivity() error { //TODO - use the neo4j connec
...
+		return Concept{}, false, nil
 	} else {
 		thing, err := mapToResponseFormat(results[0], cd.env)
 		return thing, true, err
 	}
+
 }
 
-func isContent(thng neoThing) bool {
+func isContent(thng neoConcept) bool {
```
Comment (scott-ace-newton) [inline things/cypher.go]: Is this even possible?

Comment (nwrigh) [reply things/cypher.go]: Yes we specifically have to ONLY return non content concepts and although we define content as a concept our consumer do not want this behaviour

Comment (scott-ace-newton) [reply things/cypher.go]: We dont have anything in Neo which is both a content and a concept? Nor do we have content with an UppIdentifier so not sure still unsure as to the purpose of this check

Comment (nwrigh) [reply things/cypher.go]: Actually with UPPIdentifiers you are right we don't need this but I am inclined to leave it for the moment as we are removing Identifers in the imminent future and this (along with the test of course) will aid as a reminder that we shouldn't return Content in this service

**Pull request merged**